@xen-orchestra/lite/src/App.vue

```diff
@@ -58,13 +60,16 @@ link.href = favicon;
 
 document.title = "XO Lite";
 
-if (window.localStorage?.getItem("colorMode") !== "light") {
-  document.documentElement.classList.add("dark");
-}
-
 const xenApiStore = useXenApiStore();
 const hostStore = useHostStore();
 useChartTheme();
+const uiStore = useUiStore();
+
+const { Ctrl_Shift_D } = useMagicKeys();
```

Comment:
Ctrl + Shift + D is a keyboard shortcut used by firefox on the default configuration.
I personally don't mind, as I don't use this firefox shortcut, but I think it's good to point it out.

Reply:
What about a single character instead of using modifiers? Apparently it's hard to tell which shortcuts are safe to use across OS's and browsers but a single character like "D" should be pretty safe, right? Provided that the focus is not on a text input, I don't think it should conflict with anything.